fhir-model/src/main/java/com/ibm/fhir/model/util/FHIRUtil.java

```diff
@@ -266,7 +266,33 @@ public static OperationOutcome buildOperationOutcome(String message, IssueType i
      *            the resource for which the location header value should be returned
      */
     public static URI buildLocationURI(String type, Resource resource) {
-        return URI.create(type + "/" + resource.getId() + "/_history/" + resource.getMeta().getVersionId().getValue());
+        StringBuilder sb = new StringBuilder();
+        sb.append(type);
+        sb.append("/");
+        sb.append(resource.getId());
+        sb.append("/_history/");
+        sb.append(resource.getMeta().getVersionId().getValue());
+        return URI.create(sb.toString());
+    }
+
+    /**
+     * Builds a relative "Location" header value for the specified resource type/id/version. This will be a string of the form
+     * <code>"<resource-type>/<id>/_history/<version>"</code>. Note that the server will turn this into an absolute URL prior to
```

Comment:
pretty minor but i found that javadoc strips the values in `< >` because it treats them like html tags:
```suggestion
     * <code>"[resource-type]/[id]/_history/[version]"</code>. Note that the server will turn this into an absolute URL prior to
```

Reply:
Funny - I copied this from another comment. I'll fix that too.